engine/api/workermodel/registration.go

```diff
@@ -79,9 +81,10 @@ func updateAllToCheckRegistration(db gorp.SqlExecutor) error {
 // UpdateSpawnErrorWorkerModel updates worker model error registration
 func UpdateSpawnErrorWorkerModel(db gorp.SqlExecutor, modelID int64, spawnError sdk.SpawnErrorForm) error {
 	// some times when the docker container fails to start, the docker logs is not empty but only contains utf8 null char
-	if spawnError.Error == string([]byte{0x00}) {
-		spawnError.Error = ""
-	}
+	spawnError.Error = strings.ReplaceAll(spawnError.Error, string([]byte{0x00}), "")
```

Comment:
Could you check if we can use a replacer:
https://golang.org/pkg/strings/#NewReplacer